lib/sorbet-rails/model_plugins/active_record_assoc.rb

```diff
@@ -114,9 +114,16 @@ def populate_collection_assoc_getter_setter(assoc_module_rbi, assoc_name, reflec
       return_type: relation_class,
     )
     unless assoc_should_be_untyped?(reflection)
+      if reflection.klass.table_exists?
+        # Normally the id_type is an Integer, but it could be a String if using
+        # UUIDs.
+        id_type = SorbetRails::ModelPlugins::ActiveRecordAttribute.new(reflection.klass, @available_classes).type_for_column_def(reflection.klass.columns_hash['id']).to_s
+      else
+        id_type = "T.untyped"
+      end
```

Comment:
why don't we put this method into a base module shared between the plugins, eg `model_utils.rb`?